parser.y

```diff
@@ -10771,4 +10789,127 @@ LoadStatsStmt:
 		}
 	}
 
+/********************************************************************************************
+ *
+ *  Create Sequence Statement
+ *
+ *  Example:
+ *      CREATE [OR REPLACE] [TEMPORARY] SEQUENCE [IF NOT EXISTS] sequence_name
+ *      [ INCREMENT [ BY | = ] increment ]
+ *      [ MINVALUE [=] minvalue | NO MINVALUE | NOMINVALUE ]
+ *      [ MAXVALUE [=] maxvalue | NO MAXVALUE | NOMAXVALUE ]
+ *      [ START [ WITH | = ] start ]
+ *      [ CACHE [=] cache | NOCACHE ] [ CYCLE | NOCYCLE]
+ *      [table_options]
+ ********************************************************************************************/
+
+CreateSequenceStmt:
+	"CREATE" OrReplace OptTemporary "SEQUENCE" IfNotExists TableName CreateSequenceOptionListOpt CreateTableOptionListOpt
+	{
+		$$ = &ast.CreateSequenceStmt{
+			OrReplace: $2.(bool),
+			IsTemporary: $3.(bool),
+			IfNotExists: $5.(bool),
+			Name: $6.(*ast.TableName),
+			SeqOptions: $7.([]*ast.SequenceOption),
+			TblOptions: $8.([]*ast.TableOption),
+		}
+	}
+
+CreateSequenceOptionListOpt:
+	{
+		$$ = []*ast.SequenceOption{}
+	}
+|	SequenceOptionList
+
+SequenceOptionList:
+	SequenceOption
+	{
+		$$ = []*ast.SequenceOption{$1.(*ast.SequenceOption)}
+	}
+|	SequenceOptionList SequenceOption
+	{
+		$$ = append($1.([]*ast.SequenceOption), $2.(*ast.SequenceOption))
+	}
+
+SequenceOption:
+	"INCREMENT" EqOpt SignedNum
+	{
+		$$ = &ast.SequenceOption{ Tp:ast.SequenceOptionIncrementBy, IntValue: $3.(int64),}
+	}
+|	"INCREMENT" "BY" EqOpt SignedNum
```

Comment:
```suggestion
|	"INCREMENT" "BY" SignedNum
```

The syntax `[ BY | = ]` in https://mariadb.com/kb/en/library/create-sequence/ means that we can either ignore it, choose `BY`, or choose `=`, but not both.